From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Haris Okanovic <haris.okanovic@ni.com>
Date: Wed, 27 Apr 2016 14:35:53 -0500
Subject: [PATCH] config-win32.h: Disable Win32 crypto API on Phar Lap ETS

Don't define USE_WIN32_CRYPTO on Phar Lap ETS. Crypto API in
advapi32.dll is not provided on this platform.

Upstream-Status: Inappropriate [Pharlap specific]

Updated for curl 7.72.

Signed-off-by: Richard Tollerton <rich.tollerton@ni.com>

diff --git a/lib/config-win32.h b/lib/config-win32.h
index 9e6d92296..9b9b236b5 100644
--- a/lib/config-win32.h
+++ b/lib/config-win32.h
@@ -717,7 +717,7 @@ Vista
 #endif
 
 /* Define to use the Windows crypto library. */
-#if !defined(CURL_WINDOWS_APP)
+#if !defined(CURL_WINDOWS_APP) && !defined(PHARLAP)
 #define USE_WIN32_CRYPTO
 #endif
 
-- 
2.28.0

